docs/topics/switch.rst

```diff
@@ -284,14 +284,15 @@ converted to the sequences used by ``gdal``.
 Coordinate Reference Systems
 ----------------------------
 
-The ``crs`` attribute of a Rasterio dataset object is an instance of Rasterio's
-``CRS`` class and works well with ``pyproj``.
+The :attr:`.DatasetReader.crs` attribute is an instance of Rasterio's
+:meth:`.CRS` class and works well with ``pyproj``.
 
 .. code-block:: pycon
 
-   >>> from pyproj import Proj, transform
+   >>> from pyproj import Transformer
```

Comment:
Updated pyproj example: https://pyproj4.github.io/pyproj/stable/gotchas.html#upgrading-to-pyproj-2-from-pyproj-1